packages/paypal-commerce-integration/src/paypal-commerce-ratepay/paypal-commerce-ratepay-payment-strategy.spec.ts

```diff
@@ -51,15 +52,21 @@ describe('PayPalCommerceAlternativeMethodRatePayPaymentStrategy', () => {
         paypalcommerceratepay: paypalCommerceRatePayMethodsOptions,
     };
 
+    const loadingIndicator = new LoadingIndicator();
+
     beforeEach(() => {
         paypalCommerceIntegrationService = getPayPalCommerceIntegrationServiceMock();
         paymentIntegrationService = new PaymentIntegrationServiceMock();
 
         strategy = new PaypalCommerceRatepayPaymentStrategy(
             paymentIntegrationService,
             paypalCommerceIntegrationService,
+            loadingIndicator,
         );
 
+        jest.spyOn(loadingIndicator, 'show').mockReturnValue(undefined);
+        jest.spyOn(loadingIndicator, 'hide').mockReturnValue(undefined);
```

Comment:
You don't need to return `undefined`. You can keep a `spyOn` method without `.mockReturnValue(undefined);` if you need to track the function elsewhere.

Reply:
For some reason it doesn't work without `mockReturnValue`